suites/suites_test.go

```diff
@@ -225,13 +224,11 @@ func generateEmqxBroker(name, namespace string) *v1beta1.EmqxBroker {
 		Spec: v1beta1.EmqxBrokerSpec{
 			Image:              "emqx/emqx:4.3.10",
```

Comment:
Maybe the default value of Image set as constant in the constants.go is better? Such as DEFAUT_EMQX_BROKER_IMAGE="emqx/emqx:4.3.10"